HelpCentre/QA/HelpCentre: Agent Wrap-up.md

```diff
@@ -0,0 +1,501 @@
+
+## Connections to other stacks
+
+* `HelpCentre: Intro to HelpCentre`
+* `HC: Scheduled - Callback follow up`
+
+## Global variables
+
+* `contentrepo_qa_token` used to auth api calls
+
+## Content dependencies
+
+* `mnch_onboarding_error_handling_button`
+* `plat_help_query_successful`
+* `plat_help_agent_helpful_response`
+* `plat_help_agent_unsuccessful_response`
+* `plat_help_call_back_response`
+* `plat_help_call_back_confirmation`
+* `plat_help_agent_call_back_number_confirmation`
+* `plat_help_agent_call_back_number_update`
+* `plat_help_agent_contact_number_save`
+* `plat_help_bot_to_agent_handover_waiting_room`
+
+# Init and FetchError messages
+
+```stack
+trigger(on: "MESSAGE RECEIVED") when has_only_phrase(event.message.text.body, "aiw")
+# interaction_timeout(60)
+
+card FetchError, then: CheckForNavBypass do
+  # Fetch and store the error message, so that we don't need to do it for every error card
+  search =
+    get(
+      "https://content-repo-api-qa.prk-k8s.prd-p6t.org/api/v2/pages/",
+      query: [
+        ["slug", "mnch_onboarding_error_handling_button"]
+      ],
+      headers: [["Authorization", "Token @global.settings.contentrepo_qa_token"]]
+    )
+
+  # We get the page ID and construct the URL, instead of using the `detail_url` directly, because we need the URL parameter for `get` to start with `https://`, otherwise stacks gives us an error
+  page_id = search.body.results[0].id
+
+  page =
+    get(
+      "https://content-repo-api-qa.prk-k8s.prd-p6t.org/api/v2/pages/@page_id/",
+      query: [
+        ["whatsapp", "true"]
+      ],
+      headers: [["Authorization", "Token @global.settings.contentrepo_qa_token"]]
+    )
+
+  button_error_text = page.body.body.text.value.message
+end
+
+card CheckForNavBypass when contact.navbypass == "BotToAgentHandoverWaitingRoom",
+  then: BotToAgentHandoverWait do
+  log("Bypassing Main Menu - Goto BotToAgentHandoverWaitingRoom")
+  update_contact(navbypass: "")
+end
+
+card CheckForNavBypass when contact.navbypass == "UnresolvedWhatNext", then: UnresolvedWhatNext do
+  log("Bypassing Main Menu - Goto UnresolvedWhatNext")
+  update_contact(navbypass: "")
+end
+
+card CheckForNavBypass when contact.navbypass == "AgentHelpfulResponse",
+  then: AgentHelpfulResponse do
+  log("Bypassing Main Menu - Goto AgentHelpfulResponse")
+  update_contact(navbypass: "")
+end
+
+card CheckForNavBypass when contact.navbypass == "UseAnotherNumber",
+  then: UseAnotherNumber do
+  log("Bypassing Main Menu - Goto UseAnotherNumber")
+  update_contact(navbypass: "")
+end
+
+card CheckForNavBypass, then: CheckResolution do
+  log("Going to CheckResolution")
+end
+
+card GotoMainMenu do
+  log("Going to Main Menu")
+  run_stack("7b50f9f4-b6cf-424b-8893-8fef6d0f489b")
+end
+
+card GotoSearchMyHealth do
+  log("Going to SearchMyHealth")
+  update_contact(navbypass: "SearchMyHealth")
+  run_stack("7b50f9f4-b6cf-424b-8893-8fef6d0f489b")
+end
```

Comment:
Yea at this point it will already have been cancelled. In this instance. these `navbypass` properties are set to bypass navigation in the flow we are sending them to